apple/Sources/FerrostarMapLibreUI/Theme/FerrostarTheme.swift

```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+
+///
+public protocol FerrostarTheme {
```

Comment:
Discussed on the call; will move these out (incl. the banner view) to a new `FerrostarSwiftUI` target.